test/WebSites/HtmlGenerationWebSite/Views/HtmlGeneration_Home/DuplicateAntiforgeryTokenRegistration.cshtml

```diff
@@ -0,0 +1,19 @@
+﻿@{
```

Comment:
Bet we didn't remove `Html.AntiForgeryToken()` calls when we changed the `BeginForm()` / `BeginRouteForm()` default. So that part is likely covered. This leaves the multiple `<form>`s (unlikely covered elsewhere) and use of `BeginRouteForm()` as the potentially-unique bits.